kymatio/scattering2d/backend/backend_skcuda.py

```diff
@@ -69,23 +69,25 @@ def __init__(self, pad_size, input_size, pre_pad=False):
         self.build()
 
     def build(self):
-        pad_size_tmp = self.pad_size
+        pad_size_tmp = list(self.pad_size)
 
         # This allow to handle the case where the padding is equal to the image size
-        if pad_size_tmp[0] == self.input_size[0]:
-            pad_size_tmp[0] -= 1
-            pad_size_tmp[1] -= 1
-        if pad_size_tmp[2] == self.input_size[1]:
+        # We need to take care that the order of the pad sizes is reversed with
```

Comment:
(Yeah, uh ... shouldn't we just make this more consistent? Especially given 3D. Probably axis ordering would be the easiest)

Reply:
I'm confused too. Please fix